client/templates/selectedProduct/selectedProduct.html

```diff
@@ -0,0 +1,30 @@
+<template name="selectedProduct">
+  <h1>Selected Product:</h1>
+    {{#if Template.subscriptionsReady}}
+      {{#if selectedProduct}}
+        {{#with selectedProduct}}
+          {{>product}}
+        {{/with}}
+      {{else}}
+        Sorry, no product with that code...
+      {{/if}}
+    {{else}}
+      Loading...
+    {{/if}}
+</template>
+
+<template name="product">
+  <h2>{{description}}</h2>
+  <p>Price: ${{price}}</p>
+  <p>Cost: ${{cost}}</p>
+  <p>Look Up Code: {{itemLookUpCode}}</p>
+  <p>Type: {{itemType}}</p>
+  <p>Quantity: {{quantity}}</p>
+  <p>Reorder Point: {{reorderPoint}}</p>
+  <p>Restock Level:{{restockLevel}}</p>
+  <p>Parent Item: {{parentItem}}</p>
+  <p>Extended Description: {{extendedDescription}}</p>
+  <p>Active: {{isActive}}</p>
+  <p>MSRP: {{msrp}}</p>
+  <p>Created at: {{createdAt}}</p>
```

Comment:
I still think we don't need to show when the product was created in the system. That's more of an internal thing. What's your opinion on it?